bigdecimal-reference.md

```diff
@@ -0,0 +1,568 @@
+
+This function can be used as an alternative to `-` binary operator that allows rounding the result after the
+calculation. It subtracts `rhs` from `lhs` and returns the result of such operation, applying the rounding
+based on `options` object, if given. `options` is an options bag that configures the rounding of this
+operation.
+
+- `lhs`: A `BigDecimal` value. If the value is from another type, it throws `TypeError`.
+- `rhs`: A `BigDecimal` value. If the value is from another type, it throws `TypeError`.
+- `options`: It is an object indicating how the round operation should be performed. It's the same options bag
+  object described on [BigDecimal.round](#bigdecimalroundvalue--options). If it's not given, no rounding 
+  operation will be applied, and the exact result will be returned.
+
+### BigDecimal.multiply(lhs, rhs [, options])
+
+This function can be used as an alternative to `*` binary operator that allows rounding the result after the
+calculation. It multiplies `rhs` by `lhs` and returns the result of such operation applying the rounding based
+on `options` object, if given. `options` is an options bag that configures the rounding of this operation.
+
+- `lhs`: A `BigDecimal` value. If the value is from another type, it throws `TypeError`.
+- `rhs`: A `BigDecimal` value. If the value is from another type, it throws `TypeError`.
+- `options`: It is an object indicating how the round operation should be performed. It's the same options bag
+  object described on [BigDecimal.round](#bigdecimalroundvalue--options). If it's not given, no rounding
+  operation will be applied, and the exact result will be returned.
+
+### BigDecimal.divide(lhs, rhs, options)
+
+This function is the main way to apply division using BigDecimals. It divides `lhs` by `rhs` and returns the
+result of such operation applying the rounding based on `options` object. `options` is an options
+bag that configures the rounding of this operation.
+
+- `lhs`: A `BigDecimal` value. If the value is from another type, it throws `TypeError`.
+- `rhs`: A `BigDecimal` value. If the value is from another type, it throws `TypeError`.
+- `options`: It is an object indicating how the round operation should be performed. It's the same options bag
+  object described on [BigDecimal.round](#bigdecimalroundvalue--options). If it's not given, no rounding
+  operation will be applied, and the exact result will be returned. If the result can't be represented (due to a
+  non-terminating decimal expansion), it throws `TypeError`.
+
+Different from other arithmetic operations on `BigDecimal` constructors, we require `options` for division
+because this is the only operation where some results can't be represented as a `BigDecimal` value (e.g. when
+we divide 1m by 3m) if we don't round. With the requirement to describe how we should round the result, it's
+then possible to return a correct result for any given input.
+
+### BigDecimal.reminder(lhs, rhs [, options])
```

Comment:
remainder (or we could call this "mod" though these operations are arguably different)